internal/providers/terraform/testdata/hcl_provider_test/renders_multiple_count_resources_correctly/expected.json

```diff
@@ -0,0 +1,371 @@
+{
+  "format_version": "1.0",
+  "terraform_version": "1.1.0",
+  "planned_values": {
+    "root_module": {
+      "resources": [
```

Comment:
Do we have test coverage for resources without counts? It seems like all new tests use counts, but should we check how other resources behave with new logic?

Reply:
yup, I'll add test coverage here, but also remember all the other resource Golden file tests cover the HCL functionality. So don't fret too much that it's not covered by this specific JSON file case.